pkg/store/event/event.go

```diff
@@ -35,9 +35,8 @@ func NewStore(db *dbstore.DB) core.EventStore {
 	db.AutoMigrate(&core.PodRecord{})
 
 	es := &eventStore{db}
-	if err := es.DeleteIncompleteEvents(context.Background()); err != nil && gorm.IsRecordNotFoundError(err) {
-		log.Error(err, "failed to delete all incomplete events")
-	}
```

Comment:
Because the return value of the delete function is not error, these lines were deleted.